data/changelog/rocq/2024-12-04-coq-8.20.0.md

```diff
@@ -25,4 +25,4 @@ opam install coq.8.20.0
 ```
 The source code for the release is also directly available on:
 
-* [GitHub](https://github.com/coq/coq/archive/8.20.0.tar.gz)
+* [GitHub](https://github.com/rocq-prover/rocq-prover/archive/8.20.0.tar.gz)
```

Comment:
We said it would be rocq-prover/rocq. See https://coq.zulipchat.com/#narrow/channel/237656-Coq-devs-.26-plugin-devs/topic/How.20will.20the.20coq.2Fcoq.20repo.20be.20renamed.3F/with/495518054